Lib/Composition/TestServer.cs

```diff
@@ -30,7 +31,7 @@ public TestServer(bool verbose)
 
         public ILongPollingConnectionHandler NewConnectionHandler()
         {
-            return new TestServerConnectionHandler(this);
+            return new TestServerConnectionHandler(this, SpecFilter);
```

Comment:
I would move SpecFilter from constructor into StartTest method parameter. So each run of tests could have different filter if needed in future.

Reply:
Moved.